packages/bpk-component-button/src/BpkButton-test.js

```diff
@@ -23,10 +23,8 @@ import { render } from '@testing-library/react';
 import BpkButton from './BpkButton';
 
 describe('BpkButton accessibility tests', () => {
-  it('should not have programmatically detectable accessibility issues', async () => {
-    const { container } = render(
-      <BpkButton onClick={() => {}}>My button</BpkButton>,
-    );
+  it('should not have programmatically-detectable accessibility issues', async () => {
+    const { container } = render(<BpkButton>My button</BpkButton>);
```

Comment:
The component actually renders something completely different if you provide a `href`, so I think we should test that too

Reply:
Good catch